languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22406,6 +22406,34 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- ====================================================================== -->
     <category id="COLLOQUIALISMS" name="Umgangssprache" type="register">
         <!-- TODO Make one Java-rule for context-independent, colloquial expressions and maintain a list -->
+            <rule id="DRUM_DARUM" name="drum (darum)">
+                <antipattern>
+                    <token>Drum</token>
+                    <token>und</token>
+                    <token>Dran</token>
+                </antipattern>
+                <antipattern>
+                    <token>sei</token>
+                    <token>es</token>
+                    <token>drum</token>
+                </antipattern>
+                <antipattern>
+                    <token>sei</token>
+                    <token spacebefore="no" regexp="yes">&apostrophe;</token>
+                    <token spacebefore="no">s</token>
+                    <token>drum</token>
+                </antipattern>
+                <pattern>
+                    <token>drum</token>
+                </pattern>
+                <message>'Drum' ist sehr umgangssprachlich. Vielleicht passt <suggestion>darum</suggestion> hier besser.</message>
```

Comment:
In einem formellen Kontext ist das standardsprachliche "darum" eventuell passender. (oder so)